sdk/confidentialledger/azure-security-confidentialledger/README.md

```diff
@@ -36,7 +36,7 @@ The simplest way of providing a bearer token is to use the `DefaultAzureCredenti
 
 #### Using a client certificate
 
-As an alternative to Azure Active Directory, clients may choose to use a client certificate to authenticate via mutual TLS. `CertificateCredential` may be used for this purpose.
+As an alternative to Azure Active Directory, clients may choose to use a client certificate to authenticate via mutual TLS. `CertificateCredential` may be used for this purpose. This is not the recommended approach for anyone new to the service. 
```

Comment:
Update the `azure-identity` version on line 52 to 1.5.3. We should keep this updated for each release to reflect the latest version of the package. The same goes for the version of this library on line 24. Before GA, we should change it to `1.0.0` when the pom.xml is updated to 1.0.0.